azkaban-solo-server/src/main/java/azkaban/soloserver/AzkabanSingleServer.java

```diff
@@ -86,6 +121,23 @@ public boolean implies(final ProtectionDomain domain, final Permission permissio
       return;
     }
 
+    if (props.getBoolean(Constants.ConfigurationKeys.ENABLE_QUARTZ, true)) {
```

Comment:
Change default value to false. This is not just used in unit tests; I've used it when manually starting the standalone server for manually testing this feature. Because it requires a service restart with a code change in-between, an automated unit test doesn't seem feasible. Additionally, it could easily be used by anyone hoping to use the relevant features in a standalone instance for whatever purpose. (Testing, demonstration, small scale deployment, etc)